src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -245,18 +247,33 @@ private async void ChangeStates(ControlState state)
                 // Load the team project list, show progress animation
                 try
                 {
-                    ToggleLoading(true);
                     Dispatcher.Invoke(projects.Clear);
-                    var newProjectList = await UpdateTeamProjects().ConfigureAwait(true); // need to come back to original UI thread. 
-                    newProjectList.ForEach(p => projects.Add(p));
+
+                    // If the main window is always on top, then an error occurs where
+                    //  the login dialog is not a child of the main window, so we temporarily
+                    //  turn topmost off and turn it back on after logging in
+                    bool oldTopmost = false;
```

Comment:
this code is copied from [here](https://github.com/microsoft/accessibility-insights-windows/blob/b3c5d01eb9d125ba7ebc696e406a346416324cd7/src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs#L410) because the dialog can show up when calling UpdateTeamProjects